lib/plugins/extension/script.js

```diff
@@ -116,18 +116,30 @@ jQuery(function(){
     if ( $extmgr.find('.plugins, .templates').hasClass('active') ) {
         var $extlist = jQuery('#extension__list');
         $extlist.addClass('hasDisplayOptions');
-        var $displayOpts = jQuery('<p>', { id: 'extension__viewoptions'} ).appendTo($extmgr.find( '.panelHeader' ));
 
+        var $displayOpts = jQuery('<p>', { id: 'extension__viewoptions'} ).appendTo($extmgr.find( '.panelHeader' ));
         $displayOpts.append(LANG.plugins.extension.display_viewoptions);
 
         var displayOptionsHandler = function(){
             $extlist.toggleClass( this.name );
+            DokuCookie.setValue('extension__show_'+this.name, $extlist.hasClass(this.name) ? '1' : '0');
```

Comment:
Yeah, let's shorten this to ``'ext_'+this.name`` to avoid running into cookie length problems